client/src/main/scala/org/apache/celeborn/client/WorkerStatusTracker.scala

```diff
@@ -123,17 +123,21 @@ class WorkerStatusTracker(
     }
   }
 
+  def removeFromBlacklist(workers: JHashSet[WorkerInfo]): Unit = {
+    blacklist.keySet.removeAll(workers)
```

Comment:
Why use keySet here?

Reply:
According to the doc of ConcurrentHashMap, the keySet()  returns a Set view of the keys contained in this map. The set is backed by the map, so changes to the map are reflected in the set, and vice-versa.